omnibus/package-scripts/ddot-deb/postinst

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+##########################################################################
+#             DO NOT EDIT THIS SCRIPT DIRECTLY.                          #
+#                                                                        #
+# The installation logic is handled by the installer at in the following #
+# file: pkg/fleet/installer/packages/datadog_agent_ddot_linux.go         #
+#                                                                        #
+##########################################################################
+
+INSTALL_DIR=/opt/datadog-agent
+
+# If we are inside the Docker container, do nothing
+if [ -n "$DOCKER_DD_AGENT" ]; then
+    echo "Installation from docker-dd-agent, nothing to do in postinst"
+    exit 0
+fi
+
+# Run the postinst. See pkg/fleet/installer/packages/datadog_agent_ddot_linux.go
+${INSTALL_DIR}/embedded/bin/installer postinst datadog-agent-ddot deb || true
```

Comment:
The prerm makes me think we support the case agent OCI + ddot deb. In that case the installer path may not be in /opt/datadog-agent. Maybe we could simply use the `datadog-installer` symlink?